src/gui/treeview.cpp

```diff
@@ -0,0 +1,80 @@
+#include "treeview.h"
+
+#include <QDir>
+#include <QHeaderView>
+#include <QKeyEvent>
+#include <QProcess>
+#include <QStandardPaths>
+
+namespace NeovimQt {
+TreeView::TreeView(NeovimConnector *nvim, QWidget *parent)
+: QTreeView(parent), m_nvim(nvim) {
+	model = new QFileSystemModel(this);
+
+	setModel(model);
+
+	header()->hide();
+	hide();
+	// FIXME: MAGIC NUMBER!!
```

Comment:
Whats up with this magic number comment?

Reply:
Columns contain additional information about the files, I'll change that to be more meaningful